manager/notifier.go

```diff
@@ -544,6 +545,77 @@ func (n *notifier) sendEmailNotification(to string, op notificationOp, a *Alert)
 	return writeEmailBody(wc, *smtpSender, to, status, a, n.alertmanagerURL)
 }
 
+// opGenieMessageCreate is the request for sending a opsGenie notification. We are not specifying all the
+// fields. The Details field is populated with the labels from the alert.
+type opsGenieMessageCreate struct {
+	Message     string            `json:"message"`
+	Description string            `json:"description"`
+	Alias       string            `json:"alias"`
+	Teams       []string          `json:"teams"`
+	Details     map[string]string `json:"details"`
+}
+
+// opsGenieMessageClose closes an open alert in OpsGenie
```

Comment:
[ Quoting <notifications@github.com> in "Re: [alertmanager] Alertmanager: Ad..." ]
>> @@ -544,6 +545,77 @@ func (n *notifier) sendEmailNotification(to string, op notificationOp, a *Alert)
>>  	return writeEmailBody(wc, *smtpSender, to, status, a, n.alertmanagerURL)
>>  }
>>
>> +// opGenieMessageCreate is the request for sending a opsGenie notification. We are not specifying all the
>> +// fields. The Details field is populated with the labels from the alert.
>> +type opsGenieMessageCreate struct {
>> +	Message     string            `json:"message"`
>> +	Description string            `json:"description"`
>> +	Alias       string            `json:"alias"`
>> +	Teams       []string          `json:"teams"`
>> +	Details     map[string]string `json:"details"`
>> +}
>> +
>> +// opsGenieMessageClose closes an open alert in OpsGenie
>
>Period at the end of comment.

Ack.

Note I still need to fully test this at our end. Hopefully that will happen
today.